kinto/core/authorization.py

```diff
@@ -91,7 +90,15 @@ def permits(self, context, principals, permission):
             shared = context.fetch_shared_records(permission,
                                                   principals,
                                                   self.get_bound_permissions)
-            allowed = shared is not None
+            # If allowed to create this kind of object on parent, then allow to obtain the list.
+            if len(bound_perms) > 0:
+                parent_uri = bound_perms[0][0]  # Bounds perms are ordered.
```

Comment:
This takes the first object URI of the bound perms. 
But the first is not always the parent! (Ex. record → collection → bucket)
So I guess this is wrong for records, because it will check that the user has the permission `record:create` on `/buckets/test` which will never be the case.